nodejs/week1/meal_sharing_website/src/backend/index.js

```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+const app = express();
+
+// Modular routes:
+
```

Comment:
By convention we put all the imports (requires) at the top of the file. Otherwise it is hard to understand what is going on in a file because imports are intermingled with code. So use:

```
// ------- imports/requires ---------------
// these are all the things I need in this file that
// come from other files

// -------- actual code --------
// this is the purpose of this file. Why this file exists
// it is the most relevant part of the file. That is why
// it is specially important that it is super tidy and clear
```

A nice metaphor are ingredients and a recipe. The ingredients are what you need to make something work, and the recipe are the steps you follow to create a dish using the ingredients above. The imports/requires are your ingredients. The code that you write is your recipe. Ingredients first. Then use the ingredients in your recipe :)

Reply:
Implemented feedback on all the below comments.